### PR TITLE
Adds inspect text for the title screen

### PR DESCRIPTION
**Pull request by etherware-novice**
## About The Pull Request
You can now inspect the title/splash screen for a nice little set of messages

## Why It's Good For The Game
Adds something to do while the game is loading, also more examine text is always nice

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

titleinspect

</details>

## Changelog
:cl:
add: Added inspect text for the splashscreen turf
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
can new_players even examine?

**Comment by etherware-novice** [comment]
if you right click and hit examine yea

**Comment by FortSumter** [comment]
I like Easter eggs

**Comment by etherware-novice** [comment]
same ^^

**Review by Crossedfall — CHANGES_REQUESTED** [review]
Convert your list into a string file and pick from that instead. Similar to tips and other configurable string files https://github.com/BeeStation/BeeStation-Hornet/blob/ef0cbd8ddcea727048bde01b0f44d7a3a6cd7471/code/__DEFINES/configuration.dm#L23-L42

**Review by Crossedfall — CHANGES_REQUESTED** [review]
Partway there. I linked to how I wanted you to do it. Make the file a JSON, set it as a config DEFINE, and then pull from it. Added examples below.

https://github.com/BeeStation/BeeStation-Hornet/blob/master/strings/redpill.json
https://github.com/BeeStation/BeeStation-Hornet/blob/ef0cbd8ddcea727048bde01b0f44d7a3a6cd7471/code/__DEFINES/configuration.dm#L35
https://github.com/BeeStation/BeeStation-Hornet/blob/0c500f3c39509c98a0d5c931b81c45333624148f/code/modules/reagents/reagent_containers/pill.dm#L52

File: strings/splash.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+	"splashes": ["Paved by the tears of coders.",
```
Comment (Crossedfall) [inline strings/splash.json]: Drop the first string into its own line and indent accordingly

File: strings/splash.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+	"splashes": [
+		"Paved by the tears of coders.",
+		"Built on the only engine 100% of people find unusable!",
+		"It's a miracle BYOND hasn't crashed yet.",
+		"The only 'popular' game on BYOND.",
```
Comment (PowerfulBacon) [inline strings/splash.json]: We probably don't want to get on the bad side of byond. I'd remove these ones.
The second one can be replaced with "It's a mircale it(/we) hasn't crashed yet"

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]

**Review by PowerfulBacon — DISMISSED** [review]

File: strings/splash.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+	"splashes": [
+		"Paved by the tears of coders.",
+		"It's a miracle the server hasn't crashed yet.",
+		"Now with 33% more greytide!",
+		"Featuring ian!",
```
Comment (Crossedfall) [inline strings/splash.json]:
```suggestion
		"Featuring Ian!",
```

File: strings/splash.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+		"The icon of anticipation",
+		"The only place where you can hit yourself with a toolbox on accident.",
+		"Where the Chain of Command is a Chain of Incompetence",
+		"AI LAW 2 START GAME",
```
Comment (Crossedfall) [inline strings/splash.json]:
```suggestion
		"AI LAW 2, START THE GAME",
```

File: strings/splash.json [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+		"Honk Honk! You should pick clown.",
+		"Watch for nukies!",
+		"Traitor Central!",
+		"You've definitely seen this before.",
+		"Greytide Stationwide.",
```
Comment (Crossedfall) [inline strings/splash.json]:
```suggestion
		"Greytide station wide.",
```

File: strings/splash.json [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+		"Ignore what poly says.",
```
Comment (Crossedfall) [inline strings/splash.json]:
```suggestion
		"Ignore what Poly says.",
```

File: strings/splash.json [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+		"Turn on ya damn suit sensors!",
```
Comment (Crossedfall) [inline strings/splash.json]:
```suggestion
		"Turn on your damn suit sensors!",
```

File: strings/splash.json [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+		"Celebrating 20 years of Nanotrasen.",
```
Comment (Crossedfall) [inline strings/splash.json]:
```suggestion
		"Celebrating 20+ years of Nanotrasen.",
```

File: strings/splash.json [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+		"This is supposed to be a research base?",
+		"Blame the clown.",
+		"13th time the charm.",
```
Comment (Crossedfall) [inline strings/splash.json]:
```suggestion
		"13th time's the charm.",
```

**Review by Crossedfall — CHANGES_REQUESTED** [review]
Some minor grammar stuff. Otherwise it's ready

**Review by Crossedfall — APPROVED** [review]

**Pull request merged**